react-typescript/tmpl/src/index.ts

```diff
@@ -4,7 +4,7 @@ import installExtension, { REACT_DEVELOPER_TOOLS } from 'electron-devtools-insta
 
 // Keep a global reference of the window object, if you don't, the window will
 // be closed automatically when the JavaScript object is garbage collected.
-let mainWindow;
+let mainWindow: Electron.BrowserWindow | null = null;
```

Comment:
it is not weird and expected behavior. @types/electron package is ambient module declaration and patches gloval type definitons, so if consumer code imports and consumes electron package, it automatically supports 'Electron' as global namespace for modules.